ui/app/index.html

```diff
@@ -10,6 +10,7 @@
 	<script src="lib/angular-resource.min.js"></script>
 	<script src="lib/angular-moment.min.js"></script>
 	<script src="lib/ui-bootstrap-custom-tpls-2.0.1.min.js"></script>
+	<script src="lib/ngStorage.min.js"></script>
```

Comment:
We are using local storage to store the most recent comment email address as will. There we didn't include another dependency. Are you sure we need this for the code below?